### ISSUE TITLE
Improve style and format of emails

### ISSUE THREAD

**Issue by stevensona** [stevensona/briskforms#16]
The emails lack a modern look and feel as a direct result of no styling. They also need to be responsive.

**Comment by oqx** [comment]
Howdy stevensona! I'd love to contribute to the look and feel. Let me know if you'd like to connect.

**Comment by stevensona** [comment]
@oqx, absolutely. #18 is still currently open, so I don't mind if you want to build off that or just start from scratch with it. I would love contributions of any kind, as graphic design isn't exactly my wheelhouse. Thanks! :+1:
